resources/assets/js/views/MenuEditor.vue

```diff
@@ -47,6 +47,7 @@
 							:index="index"
 							itemKey="text"
 							name="Link text"
+							placeholder="Homepage"
```

Comment:
@theninja revert revert - remove placeholder text as users think the field is already filled in